lib/smb2/packet/tree_connect_request.rb

```diff
@@ -4,13 +4,18 @@ class Smb2::Packet
   # [Section 2.2.9 SMB2 TREE_CONNECT Request](https://msdn.microsoft.com/en-us/library/cc246567.aspx)
   class TreeConnectRequest < Smb2::Packet
     nest :header, RequestHeader
-    unsigned :struct_size, 16
+    unsigned :struct_size, 16, default: 9
```

Comment:
It's not obvious why this would default at `9` without a derivation from flag until a developer reads link.  Can you add a `@note` with a quote of "StructureSize (2 bytes): The client MUST set this field to 9, indicating the size of the request structure, not including the header. The client MUST set it to this value regardless of how long Buffer[] actually is in the request being sent." like you did for the other must be set field?